### ISSUE TITLE
Simple list web component visual focus issue

### ISSUE THREAD

**Issue by PetyaMarkovaBogdanova** [nvaccess/nvda#19246]
### Brief summary

Hi!
For the product I am working on (Web components framework) we have an issue reported by customer:
In simple list web component with slotted list items, when NVDA is open, the visual focus is lost, when navigating through list items, using a keyboard.

### Steps to reproduce

You can check out this sample, created by AI tool:
https://codepen.io/petyamarkovabogdanova/pen/vEGeNoO
Steps to reproduce:
1.Open NVDA
2.Navigate to the first list item
3. Try to navigate using arrow down

### Actual behavior

Result: NVDA is announcing the next list item, but not showing the visual focus.

### Expected behavior

Expected result: When NVDA is closed the visual focus crawl through list items, when navigating through them. It should be the same with open reader.

### NVDA type

installed copy

### NVDA version

2025.2

### Windows version

Windows 11 Cloud PC

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by Adriani90** [comment]
Hello, this is intentional. NVDA uses browse mode navigation by default on websites, meaning the keyboard is navigating a virtual document (virtual version of the website) in which the screen reader can use the semantics of the website to navigate e.g. with h between headings, g between graphics, t between tables etc.
This makes navigation on a website much easier for blind people.
For this to work, NVDA uses a virtual cursor that is independent from the system focus. You can turn on a visual indicator for NVDA's browse mode virtual cursor in NvDA settings / vision.

If you want the system focus to be changed when using the keyboard, then you can switch to focus mode by doing following:
* Either press NVDA+space bar to switch between browse and focus mode
* Or disable browse mode on page load permanently in NVDA settings / browse mode.

I really recommend you to have a look at NVDA's user guide and get familliar with the software, your clients asking for accessibility will highly appreciate it.
